src/components/WebexMember/WebexMember.stories.js

```diff
@@ -18,3 +18,10 @@ StatusEnabled.args = {
   personID: 'user1',
   displayStatus: true,
 };
+
+export const MemberMuted = Template.bind({});
```

Comment:
```suggestion
export const Muted = Template.bind({});
```
Naming this story `muted` should be sufficient, it will display under the `WebexMember` group under Storybook